src/kava.js

```diff
@@ -33,6 +33,10 @@ class Kava extends BasePlugin {
   _loadStartTime: number;
   _lastDroppedFrames: number = 0;
   _lastTotalFrames: number = 0;
+  _performaceObserver: window.PerformanceObserver;
```

Comment:
typo `_performaceObserver`
```suggestion
  _performanceObserver: window.PerformanceObserver;
```

Reply:
need to fix across